example/lib/VisionDetectorViews/painters/pose_painter.dart

```diff
@@ -29,6 +29,70 @@ class PosePainter extends CustomPainter {
             1,
             paint);
       });
+
+      final leftPaint = Paint()
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = 6
+        ..color = Colors.yellow;
+
+      final rightPaint = Paint()
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = 6
+        ..color = Colors.blueAccent;
+
+      PoseLandmark leftShoulder =
+          pose.landmarks[PoseLandmarkType.leftShoulder]!;
```

Comment:
What happend when the landmark is null? could you cover that case as well?

Reply:
My understanding is: if there is a pose detected, there is no case where any landmark is null.
Even when some joints are not detected (i.e. legs are not detected as a user is showing the upper body only), the google ml kit still predicts the coordinates of the missing joints.

Reply:
ok, let me test those changes